app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -130,10 +130,7 @@ protected void onStart() {
 
         mGoogleApiClient.registerConnectionCallbacks(this);
         mGoogleApiClient.registerConnectionFailedListener(this);
-
-        if (PrefUtils.isSignedIn(this)) {
-            mGoogleApiClient.connect();
-        }
+        mGoogleApiClient.connect();
```

Comment:
You are right. And we are using `GoogleApiClient` for AppIndex API even when the user is not logged in. 

So then, what should we do? We can fix SharedPrefs thing with the login successful callback. But again, it will be weird, users will click "Skip Sign-in" and they will be signed in automatically since they recently signed in to the app.